pkg/server/apiHandlerv2.go

```diff
@@ -0,0 +1,621 @@
+// Copyright (c) 2019 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package server
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/gorilla/mux"
+	"github.com/lf-edge/adam/pkg/driver"
+	"github.com/lf-edge/adam/pkg/driver/common"
+	x509Pem "github.com/lf-edge/adam/pkg/x509"
+	"github.com/lf-edge/eve/api/go/auth"
+	"github.com/lf-edge/eve/api/go/certs"
+	"github.com/lf-edge/eve/api/go/config"
+	"github.com/lf-edge/eve/api/go/evecommon"
+	uuid "github.com/satori/go.uuid"
+)
+
+type ApiRequestv2 struct {
```

Comment:
This is an exact duplicate of `type ApiRequest`, but given that these really are distinct APIs, I can see why it makes sense to leave them as separate.

Reply:
Sure, done